app/lib/clients/twitter_client.rb

```diff
@@ -11,11 +11,26 @@ def self.authenticate
     client
   end
 
+  TwitterQuery = Struct.new(:created_at, :text, :favorite_count, :retweet_count, :reply_count)
+
+  def self.reply_count(id, client)
+    replies = client.search("to:mojotech ", since_id: id, include_entities: true)
+    .select { |res| res.in_reply_to_status_id == id }
+    replies.count
+  end
+  
   def self.latest_tweet
     client = authenticate
     # API gets count amt, then filters out rts and replies
     # This guarantees there will always be a tweet
-    tweet = client.user_timeline('mojotech', count: 5, include_rts: false, exclude_replies: true).first
-    tweet
+    latest = client.user_timeline('mojotech', count: 5, include_rts: false, exclude_replies: true, tweet_mode: 'extended').first
+
+    tweet_text = latest.attrs[:full_text]
```

Comment:
Oh derp. I didn't realize they exposed `attrs` publicly. Good call!

Reply:
Hahah yeah I stumbled upon it trying to fix some stuff.